### ISSUE TITLE
Starting services fails when installing DIY Beta 2

### ISSUE THREAD

**Issue by sglangevin** [medic/medic-webapp#1013]
Saw this occur on 3 different computers. Starting services failed and when I clicked on try again, it succeeded. Could be due to the considerably large amount of sample data in this particular version.

**Comment by browndav** [comment]
Producing new builds now for acceptance testing.

**Comment by sglangevin** [comment]
Tested new build - alpha-r50 and got Platform failed to start again. I didn't see any issue when using vmware fusion on my Mac with the new iso but when I installed the new .exe on a Windows laptop, services failed to start.

With alpha-r49 I could click try again and it started right up. That doesn't seem to have worked this time around. Let me know if there are some sort of logs I can provide here.

**Comment by abbyad** [comment]
The version that Sharon is refering to (v0.2-alpha-r50) uses `medic-os-v1.1.0-20150623-x86-virtual.iso`

**Comment by browndav** [comment]
Okay, thanks, I'll give it a try on the Windows laptop here and see if I can reproduce. Do you know roughly how long it waited before failing during this most recent test?

**Comment by sglangevin** [comment]
I'd say 1-2 minutes but that's an estimate, it might have been slightly longer.

**Comment by sglangevin** [comment]
@browndav did you get a chance to try on a Windows laptop? Both Anand and Enock have struggled to install on their Windows laptops. Enock finally was successful with alpha-r48 after clicking try again about 10 times but Anand has yet to be able to get services to start on his laptop.

**Comment by browndav** [comment]
Problem remains on a couple Windows laptops despite a four-minute timeout and a lack of apparent errors in the logs. Working on putting a timeline together to figure out if there's a specific bottleneck, or if the problem is systemic and/or outside of the VMware guest.

**Comment by browndav** [comment]
We've confirmed this to be an issue with thermal/power management on the Windows 8.1 host side. The CPU frequency scaling is inexplicably locked to 40% despite the power management settings being configured for high performance. There's some evidence that this is a Lenovo-specific driver or firmware issue.

We'll likely open additional issues to cover deployment logistics, antivirus exclusion, and/or performance issues related to the host side. Closing.
